docs/customize.md

```diff
@@ -611,6 +611,7 @@ If you are curious about my personal configuration, here it is. The color scheme
 
 ```javascript
 // Define region highlight styles
+{
```

Comment:
Ok,I have amended my last commit to indent the content between the newly added brackets